sdk/go/common/resource/properties_path.go

```diff
@@ -394,9 +394,9 @@ func (p PropertyPath) reset(old, new PropertyValue) bool {
 						}
```

Comment:
These changes were required to get linting to work. I wasn't sure if I needed to make the linter pass for code that I had not modified